src/content/concepts/dependency-graph.md

```diff
@@ -3,11 +3,17 @@ title: Dependency Graph
 sort: 9
 contributors:
   - TheLarkInn
+  - EugeneHlushko
+related:
+  - title: HTTP2 Aggresive Splitting Example
+    url: https://github.com/webpack/webpack/tree/master/examples/http2-aggressive-splitting
+  - title: webpack & HTTP/2
+    url: https://medium.com/webpack/webpack-http-2-7083ec3f3ce6
 ---
 
 Any time one file depends on another, webpack treats this as a _dependency_. This allows webpack to take non-code assets, such as images or web fonts, and also provide them as _dependencies_ for your application.
 
 When webpack processes your application, it starts from a list of modules defined on the command line or in its config file.
-Starting from these _entry points_, webpack recursively builds a _dependency graph_ that includes every module your application needs, then packages all of those modules into a small number of _bundles_ - often, just one - to be loaded by the browser.
+Starting from these [_entry points_](/concepts/entry-points/), webpack recursively builds a _dependency graph_ that includes every module your application needs. Then packages all of those modules into a small number of _bundles_ - often, just one - to be loaded by the browser.
```

Comment:
Then after a colon read really weird, what about:
`...your application needs, then bundles all...`

Reply:
hmm yeah i likes it!